### DISCUSSION TITLE
Showing political messages on technical websites

### DISCUSSION THREAD

**Discussion by magicofpolitics** [twbs/bootstrap#32353] [General]
Hi, I am opening this issue to get some comments and opinions about the appropriateness of expressing political statements in technological contexts. You are using Sass as the main language to develop Bootstrap, so you may have noticed the banner about support for the "Black Lives Matter" movement on their website (https://sass-lang.com).

I know that the US society is strongly politically polarized (more than other countries) and all the social issues turn to be political in nature moving even discussions about problems that should have nothing to do with politics, in a political context. This creates distorsions in the society itself and actions like wearing a mask to slow down the spread of a global pandemic, turns out to be implicit endorsment for one party or another. I know all this, but it doesn't bother me that much because I don't live in the US, and like me a large portion of the internet users may feel the same.

Back to the issue at heart, the banner on their website says: "Sass stands with the protesters against police violence. We encourage our users to get in the streets and join them if you can.". It is a bad choice of words: in the first sentence, it defects of generalization associating the concept of violence with the police like the value of violence was somehow intrinsic to the police officers over the world. In the second sentence it conceals an anarchic sentiment by inviting the users to join protesters in the streets. Again it doesn't matter if you endorse ideologically protests and want to pillory the police officers because of a few bad ones or not. It is about giving such opinion on a website whose purpose is to give technical documentation and most importantly on a website that is relevant for the whole world, not just USA.

You may ask why I don't discuss the issue on their github profile. In fact I [did](https://github.com/sass/sass/issues/2971) but my issue was closed without any debate. The user did not give any argument it just dispotically blocked me because asking to remove the banner was a violation of the code of conduct. It is not even clear what part of the code of conduct was violated though.
![Cattura](https://user-images.githubusercontent.com/75548998/101284290-81368280-37df-11eb-9404-11d25bb38f93.JPG)

This kind of behavior contributes to make a bad perception of the Sass community and, by extension, it undermins Bootstrap prestige and authority as being the main reason why Sass is so popular (it is the only reason I use it). You migrated from LESS to Sass, I don't doubt it was for technical reasons but still I want to hear some comments. What do you think, is it appropriate to show political messages in this context? Are you confortable with that?

**Comment by patrickhlauke** [comment]
frankly, if the Sass community want to show the banner, good on them, their decision. Coming here and whining about it, in the hope that Bootstrap would put leverage on them or whatever, is rather childish.

**Reply by magicofpolitics** [reply]
I asked for opinions, not asked to put leverage on no one. Thanks for sharing, no need to offend though.

**Reply by patrickhlauke** [reply]
please, don't insult our intelligence. "Sass did something I don't agree with. I notice you use Sass. Are YOU comfortable with it?"

**Reply by magicofpolitics** [reply]
That was the question yes. Are you confortable to search for programming syntax rules and be told to join protesters on the streets?

I expected some controversial discussion for sure, but not hostility to be honest. I won't reply you again you already expressed your opinion and I already acknowledged.
Have a nice day

**Comment by mdo** [comment]
I agree with Patrick's comments in the above thread. Locking this one outright.

Personally, I agree with Sass's opinion and applaud them for making the public statement. Black lives matter.
